core/src/main/java/io/hyperfoil/core/generators/RandomUUIDStep.java

```diff
@@ -0,0 +1,74 @@
+package io.hyperfoil.core.generators;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import org.kohsuke.MetaInfServices;
+
+import io.hyperfoil.api.config.BenchmarkDefinitionException;
+import io.hyperfoil.api.config.InitFromParam;
+import io.hyperfoil.api.config.Name;
+import io.hyperfoil.api.config.Step;
+import io.hyperfoil.api.config.StepBuilder;
+import io.hyperfoil.api.session.Access;
+import io.hyperfoil.api.session.ResourceUtilizer;
+import io.hyperfoil.api.session.Session;
+import io.hyperfoil.core.builders.BaseStepBuilder;
+import io.hyperfoil.core.session.SessionFactory;
+
+public class RandomUUIDStep implements Step, ResourceUtilizer {
+   private final Access toVar;
+
+   public RandomUUIDStep(Access toVar) {
+      this.toVar = toVar;
+   }
+
+   @Override
+   public boolean invoke(Session session) {
+      toVar.setObject(session, UUID.randomUUID().toString());
```

Comment:
Two issues here:
* `UUID.randomUUID()` uses secure random source and therefore it is unsuitable for performance tests.
* You are allocating an object just to dump it into string. Generate the `char[]` directly.

Reply:
unsuitable = while you can generate millions of pseudo-random number per second the secure source can give you kilobytes per second of random data at best.

Reply:
Created `LongFastUUID`